src/status_im/ui/screens/wallet/send/redesigned_views.cljs

```diff
@@ -0,0 +1,98 @@
+            [status-im.utils.ethereum.tokens :as tokens]
+            [status-im.utils.ethereum.core :as ethereum]
+            [taoensso.timbre :as log]
+            [reagent.core :as reagent]))
+
+(defview choose-amount-token []
+  (letsubs [{:keys [transaction modal? contact native-currency]} [:get-screen-params :wallet-choose-amount]
+            balance       [:balance]
+            prices        [:prices]
+            network       [:account/network]
+            all-tokens    [:wallet/all-tokens]
+            fiat-currency [:wallet/currency]
+            web3          [:get :web3]]
+    (let [chain (ethereum/network->chain-keyword network)]
+      [amount/render-choose-amount {:web3            web3
+                                    :balance         balance
+                                    :network         network
+                                    :chain           chain
+                                    :all-tokens      all-tokens
+                                    :modal?          modal?
+                                    :prices          prices
+                                    :native-currency native-currency
+                                    :fiat-currency   fiat-currency
+                                    :contact         contact
+                                    :transaction     transaction}])))
+
+(defview transaction-overview []
+  (letsubs [{:keys [transaction flow contact]} [:get-screen-params :wallet-txn-overview]
+            prices        [:prices]
+            network       [:account/network]
+            all-tokens    [:wallet/all-tokens]
+            fiat-currency [:wallet/currency]
+            web3          [:get :web3]]
+    (let [chain (ethereum/network->chain-keyword network)
+          native-currency (tokens/native-currency chain)
+          token (tokens/asset-for all-tokens
+                                  (ethereum/network->chain-keyword network) (:symbol transaction))]
+      [overview/render-transaction-overview {:transaction     transaction
+                                             :flow            flow
+                                             :contact         contact
+                                             :prices          prices
+                                             :network         network
+                                             :token           token
+                                             :native-currency native-currency
+                                             :fiat-currency   fiat-currency
+                                             :all-tokens      all-tokens
+                                             :chain           chain
+                                             :web3            web3}])))
+
+;; MAIN SEND TRANSACTION VIEW
+(defn- send-transaction-view [_opts]
+  (reagent/create-class
+   {:reagent-render (fn [opts] [recipient/render-choose-recipient opts])}))
```

Comment:
What's the point creating class here, why `recipient/render-choose-recipient` can't be used directly?

Reply:
historical leftover, can remove

Reply:
that's not an important thing, it's just marked as a new code that's why I'm commenting

Reply:
yeah, because it's a new file